tests/pipeline_with_param_table_test.py

```diff
@@ -644,3 +685,14 @@ def test_can_make_html_repr_with_param_table() -> None:
     pl = sl.Pipeline()
     pl.set_param_table(sl.ParamTable(int, {float: [1.0, 2.0, 3.0]}))
     assert pl._repr_html_()
+
+
+def test_set_param_series_sets_up_pipeline_so_derived_series_can_be_computed() -> None:
+    ints = [1, 2, 3]
+
+    def to_str(x: int) -> str:
+        return str(x)
+
+    pl = sl.Pipeline((to_str,))
+    pl.set_param_series(int, ints)
+    assert pl.compute(sl.Series[int, str]) == sl.Series(int, {1: '1', 2: '2', 3: '3'})
```

Comment:
Do we need a test to check if someone is trying to use `set_param_series` on a pipeline that already has a `param_table` with different columns? Or maybe that is just handled by the fact that `set_param_series` is calling `set_param_table` internally?

Reply:
Yes that is handled.